nbconvert/exporters/exporter.py

```diff
@@ -52,6 +52,8 @@ class Exporter(LoggingConfigurable):
     accompanying resources dict.
     """
 
+    enabled = Bool(True).tag(config=True)
```

Comment:
Could you add a short "doc string"/help text to this to explain how/why people would use this

Reply:
Yep, thanks, I've added a one line help, if you think it needs more explanation I can add it.